core-library/src/main/java/org/silverpeas/core/io/media/image/option/AnchoringPosition.java

```diff
@@ -35,5 +35,6 @@
   East,
   SouthWest,
   South,
-  SouthEast
+  SouthEast,
+  tile
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS00115)